### PR TITLE
fix(meeting): 2021-07-16 notes

### PR DESCRIPTION
**Pull request by wesleytodd**

### CODE REVIEW

File: meetings/2021-07-16-kickoff-notes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    * Michael advocate with keeping the focus on the JS ecosystem with the the understanding that we would later test solutions against other ecosystems
+    * Naughur suggested that JS community has biggest challenge on this front, and will be difficult to pull in all of the different ecosystems
+    * Darcy, from discussions in GitHub would want to figure out additional metadata that goes along with CVEs. Would be good if we propose what additional info would be useful (could be JS specific to start). His team is going to propose schema for what maintainers could provide to go along with CVE. They have been ideating around the area for a while and want to do something sooner than later.
+    * Wes, great to hear npm/github is prioritizing, this group should help fasciliate without ever getting in the way.
+    * Wes where are CVEs being made visible
+        * npm
+        * synk
+        * container scanners (often use OS info)
+    * Naugtur should be independant
+    *
```
Comment (mhdawson) [inline meetings/2021-07-16-kickoff-notes.md:49]:
```suggestion
```

File: meetings/2021-07-16-kickoff-notes.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    * Wes, great to hear npm/github is prioritizing, this group should help fasciliate without ever getting in the way.
+    * Wes where are CVEs being made visible
+        * npm
+        * synk
+        * container scanners (often use OS info)
+    * Naugtur should be independant
+    * 
+
+* Brainstorming ideas
+  *
```
Comment (mhdawson) [inline meetings/2021-07-16-kickoff-notes.md:52]:
```suggestion
    * We spent a while discussing different ideas
```

**Review by mhdawson — APPROVED** [review]
LGTM with a few suggestions.

**Review by dominykas — APPROVED** [review]

File: meetings/2021-07-16-kickoff-notes.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+The Chair emphasized the need to define a clear scope for the work. Discussion ensued as to whether the scope should focus on the JavaScript ecosystem or something more broadly applicable.
+
+* Scope
+    * Wes mentioned that it was suggested we should consider scope beyond just the JavaScript Ecosystem
+    * Michael advocate with keeping the focus on the JS ecosystem with the the understanding that we would later test solutions against other ecosystems
+    * Naughur suggested that JS community has biggest challenge on this front, and will be difficult to pull in all of the different ecosystems
+    * Darcy, from discussions in GitHub would want to figure out additional metadata that goes along with CVEs. Would be good if we propose what additional info would be useful (could be JS specific to start). His team is going to propose schema for what maintainers could provide to go along with CVE. They have been ideating around the area for a while and want to do something sooner than later.
+    * Wes, great to hear npm/github is prioritizing, this group should help fasciliate without ever getting in the way.
```
Comment (ljharb) [inline meetings/2021-07-16-kickoff-notes.md:43]:
```suggestion
    * Wes, great to hear npm/github is prioritizing, this group should help facilitate without ever getting in the way.
```

File: meetings/2021-07-16-kickoff-notes.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        * container scanners (often use OS info)
+    * Naugtur should be independant
+    * 
+
+* Brainstorming ideas
+  *  
+
+**Action Items:**
+ * Start document on what tools report CVEs to customers
+   * npm
```
Comment (ljharb) [inline meetings/2021-07-16-kickoff-notes.md:56]:
```suggestion
   * npm
   * github
```

**Review by ljharb — APPROVED** [review]

**Pull request merged**